doc/api/buffer.md

```diff
@@ -589,6 +589,71 @@ console.log(bufA.length);
 // 42
 ```
 
+### Class Method: Buffer.split([separator])
+
+* `separator` {Buffer} A separator that gets matched against the input buffer
+* Return: {Array} Array of Buffers
+
+`Buffer.split()` behaves similar to `String.prototype.split`, as such it matches
+an input against a source buffer, splits the source buffer, removes the input
+and returns the number of resulting buffers in an array.
```

Comment:
This never specifies whether it'll return a slice or a copy. Should be defined, and make sure the implementation matches. I don't care much either way, but copy may be more intuitive.